sotodlib/site_pipeline/util.py

```diff
@@ -9,9 +17,223 @@ class ArchivePolicy:
     def from_params(params):
         if params['type'] == 'simple':
             return ArchivePolicy(**params)
+        if params['type'] == 'directory':
+            return DirectoryArchivePolicy(**params)
+        raise ValueError('No handler for "type"="%s"' % params['type'])
 
     def __init__(self, **kwargs):
         self.filename = kwargs['filename']
 
     def get_dest(self, product_id):
+        """Returns (hdf_filename, dataset_addr).
+
+        """
         return self.filename, product_id
+
+
+class DirectoryArchivePolicy:
+    """Storage policy for stuff organized directly on the filesystem.
+
+    """
+    def __init__(self, **kwargs):
+        self.root_dir = kwargs['root_dir']
+        self.pattern = kwargs['pattern']
+
+    def get_dest(self, **kw):
+        """Returns full path to destination directory.
+
+        """
+        return os.path.join(self.root_dir, self.pattern.format(**kw))
+
+
+def parse_angle(angle, default_units='deg', desired_units='deg'):
```

Comment:
I'm not the biggest fan of just adding units in here. It seems like the type of functionality we'd want in core if we're building units into the `sotodlib` at all. Here makes me expect it to end up a bit haphazard.

Is there a reason we can't just do astropy units?